### ISSUE TITLE
AccountController does not lock account after x number of login failures...

### ISSUE THREAD

**Issue by mgmccarthy** [HTBox/allReady#1134]
not too sure if this is desired behavior or not, but I came upon this comment in AccountController:

```
public async Task<IActionResult> Login(LoginViewModel model, string returnUrl = null)
{
    ...
    // This doesn't count login failures towards account lockout
    // To enable password failures to trigger account lockout, set lockoutOnFailure: true
    ...
```

do we want to enable account lockout for AllReady? If not, this comment should be deleted

**Comment by stevejgordon** [comment]
It's a good security practice to do something to avoid brute force attacks so I'd agree this would be a good idea. I can't think of any real downsides since we can set the account to unlock after a given period of time.

**Comment by mgmccarthy** [comment]
@stevejgordon if this was prioritized or talked about in today's standup, I'd be happy to start working on it.

**Comment by stevejgordon** [comment]
@mgmccarthy Not discussed I'm afraid. It might be that it falls lower as the main milestones for December will be functionality towards the v1 release.

That said I don't think it's a complex change since we're just leveraging built in Identity stuff really.

@tonysurma Do you think this would be a good security feature to have in place ready for v1?

**Comment by mgmccarthy** [comment]
@tonysurma @stevejgordon 

the real question here is if we decide to implement this policy, who does the user contact to unlock their account? That answer should determine whether or not we even need this.

**Comment by MisterJames** [comment]
**Decision** There are two parts to this, and the second will be created in a secondary issue.  We should proceed and enable this.

**Comment by MisterJames** [comment]
This issue would be completed by locking users out after X attempts (perhaps stored in config with a default value of 3).  Locked out users should see a screen letting them know they can contact the organization administrator.

**Comment by MisterJames** [comment]
Closed via #2311.
